packages/i18n/src/locales/hi/translations.ts

```
@@ -0,0 +1,813 @@
+    "आपके सब्सक्राइब किए गए कार्य आइटमों पर अपडेट रहें। सूचना पाने के लिए इसे सक्षम करें।",
+  email_notification_setting_updated_successfully: "ईमेल सूचना सेटिंग सफलतापूर्वक अपडेट हुई",
+  failed_to_update_email_notification_setting: "ईमेल सूचना सेटिंग अपडेट करने में विफल",
+  notify_me_when: "मुझे सूचित करें जब",
+  someone_mentions_me: "कोई मुझे उल्लेख करे",
+  someone_assigns_me_a_work_item: "कोई मुझे कार्य आइटम असाइन करे",
+  work_item_is_updated: "कार्य आइटम अपडेट हो",
+  someone_comments_on_my_subscribed_work_item: "कोई मेरे सब्सक्राइब किए गए कार्य आइटम पर टिप्पणी करे",
+  work_item_activity: "कार्य आइटम गतिविधि",
+  work_item_updates: "कार्य आइटम अपडेट",
+  work_items: {
+    label: "कार्य आइटम",
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)